meson.build

```diff
@@ -4,7 +4,7 @@ project(
   # Note that the git commit hash cannot be added dynamically here (it is added
   # in the dynamically generated and installed `scipy/version.py` though - see
   # tools/version_utils.py
-  version: '1.10.0',
+  version: '1.10.1',
```

Comment:
`'1.10.1.dev0'` here

Reply:
has this changed from previous approach?

- https://github.com/scipy/scipy/pull/17263
- https://github.com/scipy/scipy/pull/17176

Reply:
No, I just missed it in reviewing there. The `setup.py` equivalent uses `ISRELEASED = False` which also adds `.dev0` to the version string. And in `main` it has the same `.dev0` in `meson.build`.